### ISSUE TITLE
Unable to install provider with new Lucid version

### ISSUE THREAD

**Issue by hlozancic** [chirgjin/adonisjs-select-related#1]
Provider is not working with Lucid 17+.

```
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR! 
npm ERR! While resolving: project-api@1.0.0
npm ERR! Found: @adonisjs/lucid@17.2.0
npm ERR! node_modules/@adonisjs/lucid
npm ERR!   @adonisjs/lucid@"^17.2.0" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer @adonisjs/lucid@"^16.2.0" from adonisjs-select-related@1.0.2
npm ERR! node_modules/adonisjs-select-related
npm ERR!   adonisjs-select-related@"*" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR! 
```

**Comment by chirgjin** [comment]
Hi @hlozancic 
I've published a new version 1.0.3. Can you try using that?

**Comment by chirgjin** [comment]
Closing due to inactivity
